pydis_site/apps/api/models/bot/whitelist.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+
+from pydis_site.apps.api.models.utils import ModelReprMixin
+
+
+class Whitelist(ModelReprMixin, models.Model):
+    """Whitelisted item for a type."""
+
+    TYPE_CHOICES = (
+        ("invite", "invite"),
+        ("extension", "extension"),
```

Comment:
This should be renamed to more clearly express that it's a file extension and not, say, a Discord extension. My suggestion: `("file_ext", "File Extension"),`